src/handlers/shared/start.ts

```diff
@@ -141,12 +143,19 @@ async function handleTaskLimitChecks(username: string, context: Context, maxConc
   // check for max and enforce max
 
   if (Math.abs(assignedIssues.length - openedPullRequests.length) >= maxConcurrentTasks) {
-    const log = logger.error(username === sender ? "You have reached your max task limit" : `${username} has reached their max task limit`, {
-      assignedIssues: assignedIssues.length,
-      openedPullRequests: openedPullRequests.length,
-      maxConcurrentTasks,
-    });
-    await addCommentToIssue(context, log?.logMessage.diff as string);
+    if (username !== sender) {
+      const logMessage = `${username} has reached their max task limit`;
+      const log = logger.error(logMessage, {
+        assignedIssues: assignedIssues.length,
+        openedPullRequests: openedPullRequests.length,
+        maxConcurrentTasks,
+      });
+    
+      if (log?.logMessage?.diff) {
```

Comment:
We shouldn't need to optionally access the `LogReturn` object anymore, I double checked your logger version and it's up to date so this shouldn't be needed